go.mod

```diff
@@ -13,10 +13,12 @@ require (
 	github.com/ipfs/go-ipld-cbor v0.0.3
 	github.com/ipfs/go-ipld-format v0.2.0
 	github.com/ipfs/go-ipld-legacy v0.0.0-20210325015318-9799f2cffab0
-	github.com/ipld/go-codec-dagpb v1.2.0
+	github.com/ipld/go-codec-dagpb v1.2.1-0.20210330082435-8ec6b0fbad18
```

Comment:
ℹ️  bump / point to a release before merging

Reply:
There's at least one more change in dagpb that I need to do before the entire ipfs+merkledag is ready (the spec change), so I'd rather delay the version bump until then.